res/css/_animations.scss

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * React Transition Group animations are prefixed with 'mx_rtg--' so that we
+ * know they should not be used anywhere outside of React Transition Groups.
+*/
+
+.mx_rtg--fade {
+    &-enter {
```

Comment:
Even though it will end up looking a bit more verbose, I usually think that it's a good idea to write the full class name for both readibility and searchability across the codebase

It takes a bit of effort to understand that line 25 will be converted to `mx_rtg--fade-enter-active` and that breaks "CMD+F" that I personally rely on quite heavily

Reply:
@gsouquet I couldn't figure out the convention in our codebase, I've talked with @SimonBrandner about using the `&--` BEM convention, but I think I fully agree with `grep`ability here.